apps/studio/data/edge-functions/edge-functions-update-mutation.ts

```diff
@@ -23,10 +21,15 @@ export async function updateEdgeFunction({
 }: EdgeFunctionsUpdateVariables) {
   if (!projectRef) throw new Error('projectRef is required')
 
-  const response = await patch(`${API_ADMIN_URL}/projects/${projectRef}/functions/${slug}`, payload)
-  if (response.error) throw response.error
+  const { data, error } = await patch(`/v1/projects/{ref}/functions/{function_slug}`, {
```

Comment:
invalidation below needs to invalidate the list query as well
e.g if i rename the edge function

Reply:
Nice catch! Done!